platform_versioned_docs/version-23.4/cli/commands.mdx

```diff
@@ -0,0 +1,639 @@
+  WT_REP1,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357071_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357071_2.fastq.gz,auto
+  WT_REP2,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357072_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357072_2.fastq.gz,reverse
+  RAP1_UNINDUCED_REP1,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357073_1.fastq.gz,,reverse
+  RAP1_UNINDUCED_REP2,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357074_1.fastq.gz,,reverse
+  RAP1_UNINDUCED_REP2,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357075_1.fastq.gz,,reverse
+  RAP1_IAA_30M_REP1,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357076_1.fastq.gz,https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/testdata/GSE110004/SRR6357076_2.fastq.gz,reverse
+  ```
+
+</details>  
+<details>
+  <summary>**Pipelines**</summary>
+
+  Run `tw pipelines -h` to view the list of supported operations.
+
+  Run `tw pipelines add -h` to view the required and optional fields for adding your pipeline.
+
+  Pipelines define pre-configured workflows in a workspace. A pipeline consists of a workflow repository, launch parameters, and a compute environment.
+
+  ### Add a pipeline
+
+  Add a pre-configured pipeline to the Launchpad:
+
+  ```console
+  $ tw pipelines add --name=my_rnaseq_nf_pipeline --params-file=my_rnaseq_nf_pipeline_params.yaml https://github.com/nextflow-io/rnaseq-nf
+
+  New pipeline 'my_rnaseq_nf_pipeline' added at user workspace
+  ```
+
+  The optional `--params-file` flag is used to pass a set of default parameters that will be associated with the pipeline in the Launchpad.
+
+  :::tip
+  The `params-file` must be a YAML or JSON file using [Nextflow configuration file][nextflow-config] syntax.
+  :::
+
+  ### Import and export a pipeline
+
+  Export the configuration details of a pipeline in JSON format for scripting and reproducibility purposes.
+
+  ```console
+  $ tw pipelines export --name=my_rnaseq_nf_pipeline my_rnaseq_nf_pipeline_v1.json
+
+    Pipeline exported into 'my_rnaseq_nf_pipeline_v1.json' 
+  ```
+
+  Similarly, a pipeline can be imported to a workspace from a previously exported JSON file.
+
+  ```console
+  $ tw pipelines import --name=my_rnaseq_nf_pipeline_v1 ./my_rnaseq_nf_pipeline_v1.json
+
+    New pipeline 'my_rnaseq_nf_pipeline_v1' added at user workspace
+  ```
+
+  ### Update a pipeline
+
+  The default launch parameters can be changed with the `update` command:
+
+  ```console
+  tw pipelines update --name=my_rnaseq_nf_pipeline --params-file=my_rnaseq_nf_pipeline_params_2.yaml
+  ```
+
```

Comment:
"Launch pipelines" is a new section, right? It needs a `<details>` block and `<summary>` header?